src/constants.ts

```diff
@@ -0,0 +1,5 @@
+
+export const relayBaseUrl = 'https://api.relay.link'
+
+export const baseRelayChainId = 8453
+export const baseChainId = 'eip155:8453'
```

Comment:
Turbo nitpick since we're probably never touching this again but for the sake of OCD: can we name these two with diff vernacular?

i.e either
- match the `eth_chainId` vernacular from the JSON-RPC spec for the former (and remove relay mention while at it)
- don't call the latter `chainId` since `chainId` is an us terminology and non-standard way to refer to CAIP-2s

```suggestion
// base
export const eth_chainId = 8453
export const baseCaip2 = 'eip155:8453'
```

Reply:
Applied the first suggested changes in [056ea41](https://github.com/shapeshift/relay-claiming-cli/pull/1/commits/056ea41c5ba3e13a6e20689f0f2bf9eb5ce2af5d)